state-of-the-docs-2015.rst

```diff
@@ -0,0 +1,197 @@
+.. post:: Sep 7, 2015
+   :tags: report
+   :author: Anthony
+
+State of the Docs
+=================
+
+August has historical significance for Read the Docs, so it seemed fitting to
+wrap up August by taking a moment to step back and reflect on what we've done in
+the last year.
+
+*What makes August so significant for us?*
+
+For starters, this August saw the **5th birthday of Read the Docs**!  We're
```

Comment:
This should probably link to the initial release post: http://ericholscher.com/blog/2010/aug/16/announcing-read-docs/